### ISSUE TITLE
Not able to find a InputFile with R/tests/testthat.R 

### ISSUE THREAD

**Issue by nicolasfguillaume** [Merck/sonar-r-plugin#17]
The plugin is able to find source files, not able to find test files:

`ERROR: Not able to find a InputFile with R/tests/testthat.R `

However, in the logs I can see that this test file was indexed correctly:

`'R/tests/testthat.R' indexed as test with language 'r' `

Also, I made sure to define the properties in `sonar-project.properties`:

```
sonar.sources=R/R/
sonar.tests=R/tests/
```

I can see in the function `processIssue` in file https://github.com/Merck/sonar-r-plugin/blob/master/src/main/java/com/msd/gin/common/sonar/LintRSensor.java#L97 that the InputFile type expected is Main (and not Test):
```
InputFile.Type.MAIN
```
Which makes me think that the Test files are excluded?

This would be surprising because there are test files in the json file in the sample-project (https://github.com/Merck/sonar-r-plugin/blob/master/sample-project/lintr_out.json#L390).

Please advise.

**Comment by nicolasfguillaume** [comment]
Yes in my opinion it would make sense to lint the test files as well as the main files. 
Here is my suggestion (please note that I am not a java expert):
```
        InputFile inputFile = fileSystem.inputFile(
                fileSystem.predicates().and(
                        fileSystem.predicates().hasRelativePath(issue.getFilename()),
                        fileSystem.predicates().or(
                                fileSystem.predicates().hasType(InputFile.Type.MAIN),
                                fileSystem.predicates().hasType(InputFile.Type.TEST))));
```
Thank you.

**Comment by nicolasfguillaume** [comment]
Hello @kmoco2am could we please add these modifications to the project if they look ok to you? Thank you!

**Comment by kmoco2am** [comment]
Hi, it would may be make sense to make it configurable. There can be projects where developers care about production files quality mainly and tests could be more relaxed so excluded. What do you think?
